### PR TITLE
CVE-2024-3727 oadp-1.4 GO-2024-2842: bump github.com/containers/image…

### PR DESCRIPTION
**Pull request by weshayutin**
CVE-2024-3727 oadp-1.4 GO-2024-2842: bump github.com/containers/image/v5 v5.30.2

### CODE REVIEW

**Comment by weshayutin** [comment]
note:
go mod tidy
go: finding module for package github.com/migtools/udistribution/pkg/image/udistribution
go: finding module for package github.com/moby/docker-image-spec/specs-go/v1
go: found github.com/migtools/udistribution/pkg/image/udistribution in github.com/migtools/udistribution v0.0.6
go: found github.com/moby/docker-image-spec/specs-go/v1 in github.com/moby/docker-image-spec v1.3.1
go: github.com/konveyor/openshift-velero-plugin/velero-plugins/common imports
	github.com/migtools/udistribution/pkg/image/udistribution: github.com/migtools/udistribution@v0.0.6: parsing go.mod:
	module declares its path as: github.com/kaovilai/udistribution
	        but was required as: github.com/migtools/udistribution

File: go.mod [hunk 1]
```diff
@@ -4,15 +4,14 @@ go 1.19
 
 require (
 	github.com/bombsimon/logrusr/v3 v3.0.0
-	github.com/containers/image/v5 v5.27.0
-	github.com/go-logr/logr v1.2.4
+	github.com/containers/image/v5 v5.29.5
+	github.com/go-logr/logr v1.4.1
 	github.com/google/go-cmp v0.6.0
 	github.com/hashicorp/go-hclog v1.0.0 // indirect
-	github.com/migtools/udistribution v0.0.13-oadp-1.3
```
Comment (kaovilai) [inline go.mod:11]: How is this dependency being replaced/updated with?

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (kaovilai) [reply go.mod:11]: Why don't we run
```
git reset --hard upstream/oadp-1.4 && go get github.com/containers/image/v5@v5.30.2 && go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp@v1.19.0 && go mod tidy
```
here and this should be good.

Comment (weshayutin) [reply go.mod:11]: is the migtools/udistribution something we need always manually put back in?   It's removed after running `go get github.com/containers/image/v5`  

happy to put it back

Comment (weshayutin) [reply go.mod:11]: k. see your suggestion.. thank you

Comment (weshayutin) [reply go.mod:11]: ran `git reset --hard upstream/oadp-1.4 && go get github.com/containers/image/v5@v5.30.2 && go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp@v1.19.0 && go mod tidy`

got the same result.. I'll bow out and let someone w/ more chops play here.

**Comment by kaovilai** [comment]
Had to do essentially
```
GOTOOLCHAIN=go1.19 go get github.com/containers/image/v5@v5.30.2 && go get github.com/migtools/udistribution@v0.0.13-oadp-1.3 && go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp@v1.19.0 && go mod tidy;
```

lets see if unit test passes.

**Review by kaovilai — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**